router-tests/cache_warmup_test.go

```diff
@@ -81,20 +83,107 @@ func TestCacheWarmup(t *testing.T) {
 				})
 				require.Equal(t, `{"data":{"employees":[{"id":1,"details":{"forename":"Jens","surname":"Neuse"}},{"id":2,"details":{"forename":"Dustin","surname":"Deus"}},{"id":3,"details":{"forename":"Stefan","surname":"Avram"}},{"id":4,"details":{"forename":"Björn","surname":"Schwenzer"}},{"id":5,"details":{"forename":"Sergiy","surname":"Petrunin"}},{"id":7,"details":{"forename":"Suvij","surname":"Surya"}},{"id":8,"details":{"forename":"Nithin","surname":"Kumar"}},{"id":10,"details":{"forename":"Eelco","surname":"Wiersma"}},{"id":11,"details":{"forename":"Alexandra","surname":"Neuse"}},{"id":12,"details":{"forename":"David","surname":"Stutt"}}]}}`, res.Body)
 
+				// employee operation is warmed up from the query with the inline value 2 for the id
+
+				// variable called id
 				res = xEnv.MakeGraphQLRequestOK(testenv.GraphQLRequest{
 					Query:     `query m($id: Int!){ employee(id: $id) { id details { forename surname } } }`,
 					Variables: []byte(`{"id": 1}`),
 				})
 				require.Equal(t, "HIT", res.Response.Header.Get("x-wg-execution-plan-cache"))
 				require.Equal(t, `{"data":{"employee":{"id":1,"details":{"forename":"Jens","surname":"Neuse"}}}}`, res.Body)
 
+				// variable called jensID
+				res = xEnv.MakeGraphQLRequestOK(testenv.GraphQLRequest{
+					Query:     `query m($jensID: Int!){ employee(id: $jensID) { id details { forename surname } } }`,
```

Comment:
Can you add one more test with one query and a custom variable name like `$jensID` to verify that we're sending `$jensID` to the Subgraph instead of `$a`?